src/Nethermind/Nethermind.Consensus.AuRa/AuRaBlockProcessor.cs

```diff
@@ -113,10 +116,33 @@ private void ValidateGasLimit(BlockHeader header, BlockHeader parentHeader)
 
         private void ValidateTxs(Block block, BlockHeader parentHeader)
         {
+            (bool Allowed, string Reason)? TryRecoverSenderAddress(Transaction tx)
+            {
+                if (tx.Signature != null)
+                {
+                    IReleaseSpec spec = _specProvider.GetSpec(block.Number);
```

Comment:
can you use RecoveryStep here?

Reply:
Would have to modify recoverystep to support this. Lets leave it like it is, I think this is not the issue and we will be able to remove this code later.